packages/manager/apps/dedicated/client/app/account/kyc-documents/dashboard/constants.js

```diff
@@ -1,5 +1,12 @@
 export const MAXIMUM_SIZE = 10000000;
 export const MAXIMUM_DOCUMENTS = 10;
+export const TRACKING_DOCUMENTS = 'dedicated::account::my-documents';
+export const TRACKING_CTA_SEND_DOCUMENTS = `${TRACKING_DOCUMENTS}::upload`;
+export const TRACKING_CTA_CONFIRM = `${TRACKING_DOCUMENTS}::upload-confirm`;
+export const TRACKING_CTA_CANCEL = `${TRACKING_DOCUMENTS}::upload-cancel`;
+export const TRACKING_UPLOAD_SUCCESS = `${TRACKING_DOCUMENTS}::upload-success`;
+export const TRACKING_UPLOAD_ERROR = `${TRACKING_DOCUMENTS}::upload-error`;
+export const TRACKING_CTA_DASHBOARD = `${TRACKING_DOCUMENTS}::goto-hub`;
```

Comment:
**niptick** : to avoid having a lot of imports and exports, you should group all your tracking constants in a global object with nested values
```suggestion
export const TRACKING = {
  DOCUMENTS: 'dedicated::account::my-documents',
  CTA_SEND_DOCUMENTS: `${TRACKING_DOCUMENTS}::upload`,
  CTA_CONFIRM: `${TRACKING_DOCUMENTS}::upload-confirm`,
  CTA_CANCEL:  `${TRACKING_DOCUMENTS}::upload-cancel`,
  UPLOAD_SUCCESS: `${TRACKING_DOCUMENTS}::upload-success`,
  UPLOAD_ERROR: `${TRACKING_DOCUMENTS}::upload-error`,
  CTA_DASHBOARD: `${TRACKING_DOCUMENTS}::goto-hub`,
}
```